W-18366625 multimodal support (#71)
mulesoft/docs-idp: modules/ROOT/pages/_partials/document-action.adoc

* *Model*
+

> Select the predictive Large Language Model (LLM) to use when executing this document action. 

Author: ```suggestion
Select the predictive large language model (LLM) to use when executing this document action. 
```

* *Enable PII Masking*
+
Mask any personally identifiable information from your documents before sending it to the predictive model. Your Salesforce Organization Preferences override this setting. For more information, see https://help.salesforce.com/s/articleView?id=ai.generative_ai_trust_layer.htm&type=5[Einstein Trust Layer].

> +
> You can't enable PII masking and image recognition at the same time as they are mutually exclusive. 
> +
> [IMPORTANT]
> Enabling PII masking reduces the limit of characters allowed per request to 120,000. See xref:quotas-and-limits#prompt-limits[Prompt Limits] for additional details. 
> * *Enable Image Recognition* 
> +
> Enables the model to read and interpret images. 
> +
> You can't enable PII masking and image recognition at the same time as they are mutually exclusive. 
> +
> [WARNING]
> Using Image Recognition might increase the number of Einstein credits consumed depending on the selected model. To better understand credit usage and consumption, contact your account executive. 

Author: ```suggestion
+
You can't enable PII masking and image recognition simultaneously because they're mutually exclusive. 
+
[IMPORTANT]
Enabling PII masking reduces the limit of characters allowed per request to 120,000. See xref:quotas-and-limits#prompt-limits[Prompt Limits] for additional details. 
* *Enable Image Recognition* 
+
Enables the model to read and interpret images. 
+
You can't enable PII masking and image recognition simultaneously because they're mutually exclusive. 
+
[WARNING]
Using image recognition might increase the number of Einstein credits consumed, depending on the selected model. To better understand credit usage and consumption, contact your account executive. 
```
